packages/esm-ward-app/src/beds/bed-share-divider.component.tsx

```diff
@@ -0,0 +1,22 @@
+import { Tag } from '@carbon/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './bed-share-divider.scss';
+import { SkeletonText } from '@carbon/react';
```

Comment:
```suggestion
import { Skeleton, Tag } from '@carbon/react';
import React from 'react';
import { useTranslation } from 'react-i18next';
import styles from './bed-share-divider.scss';
```

Let's consolidate the carbon imports.